lib/readline.js

```diff
@@ -105,7 +105,7 @@ function Interface(input, output, completer, terminal) {
   }
 
   if (typeof historySize !== 'number' ||
-      isNaN(historySize) ||
+      Number.isNaN(historySize) ||
```

Comment:
I think the `typeof historySize !== 'number'` check is redundant if `Number.isNaN` is being used.